sys/include/net/ng_rpl.h

```diff
@@ -1,5 +1,6 @@
 /*
- * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ * Copyright (C) 2013 - 2014  INRIA.
+ * Copyright (C) 2015 Cenk Gündoğan <cnkgndgn@gmail.com>
```

Comment:
This change looks somehow weird, but I guess it's just Github displaying this weird or why do you replace Martine by INRIA?